templates/blog/post_detail.html

```diff
@@ -0,0 +1,46 @@
+{% extends 'base.html' %}

```

Comment:
Ensure the template name is corrected to 'blog/post-detail.html' to match the previous feedback.